Automattic/vip
# Dev-env: Allow --multisite to accept boolean, subdirectory, or subdomain values

## Description

Allows for passing in `subdomain` or `subdirectory` into the `--multisite` parameter.  

## Steps to Test

1) `npm run build`
2) Create new sites with various command line `multisite` values:

```
./dist/bin/vip-dev-env-create.js --slug test1 --multisite subdomain --elasticsearch false --mailhog false --phpmyadmin false --wordpress 6.2 --app-code ~/vipdev/vip-go-skeleton --php 8.0 --xdebug false --title Test
```
```
./dist/bin/vip-dev-env-create.js --slug test2 --multisite subdirectory --elasticsearch false --mailhog false --phpmyadmin false --wordpress 6.2 --app-code ~/vipdev/vip-go-skeleton --php 8.0 --xdebug false --title Test
```
```
./dist/bin/vip-dev-env-create.js --slug test3 --multisite true --elasticsearch false --mailhog false --phpmyadmin false --wordpress 6.2 --app-code ~/vipdev/vip-go-skeleton --php 8.0 --xdebug false --title Test
```
```
./dist/bin/vip-dev-env-create.js --slug test4 --multisite false --elasticsearch false --mailhog false --phpmyadmin false --wordpress 6.2 --app-code ~/vipdev/vip-go-skeleton --php 8.0 --xdebug false --title Test
```
```
./dist/bin/vip-dev-env-create.js --slug test5 --multisite invalid --elasticsearch false --mailhog false --phpmyadmin false --wordpress 6.2 --app-code ~/vipdev/vip-go-skeleton --php 8.0 --xdebug false --title Test
```
Ensure values in the `instance_data.json` are valid (boolean, `"subdirectory"`, `"subdomain"`)
3) Repeat step 2 with prompt
